src/components/formStepper/FormStepper.tsx

```diff
@@ -67,7 +67,10 @@ const FormStepper = (props: Props): React.ReactElement => {
 
   // scroll down to ensure submit notification and button to main page are visible
   useEffect(() => {
-    mainPageButtonRef.current?.scrollIntoView({ behavior: 'smooth' });
+    mainPageButtonRef.current?.scrollIntoView({
+      behavior: 'smooth',
+      block: 'end'
```

Comment:
```suggestion
      block: 'nearest'
```
I think this would make the behavior smoother, so it wouldn't scroll unnecessarily